SU-SWS/decanter
# D8CORE-1396: making sure the mobile menu does not go under the hp hero

# READY FOR REVIEW

# Summary
- To prevent the open mobile menu at the medium grid size from going underneath the homepage hero card.

# Needed By (Date)
- 2/21

# Urgency
- med

# Steps to Test

1. Pull in the branch.
2. Build and clear cache.
3. Open a browser window in the medium grid size so you have the hamburger menu.
4. Open the mobile menu and verify it is on top of the hero card.

# Affected Projects or Products
- Decanter and D8CORE project.

# Associated Issues and/or People
- D8CORE-1396
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
